### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lenemter**
* Avoid syncing animations (not sure how they might get unsynced)
* Avoid using actions
* Get rid of `index_delta` and `next_delta` stuff
* Cleanup keypress handling

### CODE REVIEW

File: src/Cards/UserCard.vala [hunk 1]
```diff
@@ -236,20 +235,10 @@ public class Greeter.UserCard : Greeter.BaseCard {
             }
         });
 
-        // This makes all the animations synchonous
-        form_revealer.size_allocate.connect ((alloc) => {
```
Comment (danirabbit) [inline src/Cards/UserCard.vala:240]: I think originally this was meant to sync the animation of the entry being revealed with the animation of the card switching. But it seems like maybe we broke that. Not sure if it's worth trying to fix instead?

**Review by danirabbit — CHANGES_REQUESTED** [review]
Can you resolve conflicts please? :)

File: src/Cards/UserCard.vala (hunk 1 above)
Comment (lenemter) [reply src/Cards/UserCard.vala:240]: I think it's a tricky thing to do with Gtk.Revealer + Hdy.Carousel. Maybe worth looking into this after Gtk4 port?

**Comment by lenemter** [comment]
@danirabbit Done

**Review by danirabbit — APPROVED** [review]
:rocket:

**Pull request merged**